File: 11-2.cpp
#include <opencv2/opencv.hpp>

using namespace cv;

int main(){
    Mat img_gary = imread("test.png", IMREAD_GRAYSCALE);

    Mat kernel = getStructuringElement(MORPH_RECT, Size(3,3));
    Mat img_result;
    dilate(img_gary, img_result, kernel, Point(-1,-1), 1);
    imshow("Input", img_gary);
    imshow("Result", img_result);
    waitKey(0);

    destroyAllWindows();

    return 0;
}

File: 11-2.py
import cv2 as cv

img_gray = cv.imread('test.png', cv.IMREAD_GRAYSCALE)

kernel = cv.getStructuringElement(cv.MORPH_RECT, (3,3))
img_result = cv.dilate(img_gray, kernel, 1)

cv.imshow('Input', img_gray)
cv.imshow('Result', img_result)

cv.waitKey(0)
cv.destroyAllWindows()